### DISCUSSION TITLE
Where to find the layout for books and chapters

### DISCUSSION THREAD

**Discussion by aducom** [e107inc/e107#4985] [Q&A Core]
I'm trying to change the layout of the books. In the Grid template, the header has a link to the chapters but is in a large font, the detail text is shown with centered-text that I would like to change. I use the bootstrap 5 theme. I have done my best, but I cannot figure out where the templates are to change this. Not in the theme folder imho... Anybody?

**Comment by Alex-e107nl** [comment]
Ola, you can find them in the e107_core/templates, you can make a copy in the theme folder if you want to... (look at bs3 theme)

**Comment by aducom** [comment]
Thanks Alex, will try.

**Reply by aducom** [reply]
Yes, solved it. Again thnx Alex.
